portal/templates/portal/teach/materials.html

```diff
@@ -282,6 +288,75 @@ <h3>Assessments <span class="ks-tag">Upper KS2</span></h3>
     </div>
 </div>
 
+<div id="ks3">
+    <div class="banner banner--aimmo-materials row">
+        <section>
+            <div class="row row--under">
+        <img title="AIMMO Logo" alt="AIMMO icon" src='{% static "portal/img/aimmo_logo.png" %}'>
+            </div>
+            <div class="row row--under">
+        <img title="Preview Version" alt="Preview Version" src='{% static "portal/img/preview_version_label.png" %}'>
+            </div>
+        <p>The Key Stage 3 teaching resources and AI:MMO game are still being developed.</p>
+        </section>
+    </div>
+    <div class="background col-sm-8 col-center">
+        <section>
+            <h2>Key Stage 3</h2>
+            <img title="KS3" alt="KS3 icon" src='{% static "portal/img/stage_icon_ks3.png" %}'>
+            <h4>Playing AI:MMO will build on skills learnt in Rapid Router (Key Stages 1 and 2).
+                The game is a Massively Multiplayer Online (MMO) time travelling adventure, which will take them to the
+            next level on their coding journey: Artificial Intelligence and Python.</h4>
+            <a href="{{ 'KS3/KS3_resources.zip'|cloud_storage }}" class="button button--regular button--secondary--aimmo--download">Download all KS3 resources</a>
+            <img title="ZIP" alt="ZIP icon" src='{% static "portal/img/icon_zip.png" %}'>
+        </section>
+        <h3>Tutorial level sessions <span class="ks-tag">KS3</span></h3>
+        <p>Coming soon.</p>
+        <h3>Resource Sheets <span class="ks-tag">KS3</span></h3>
+        <p>Resource sheets help you teach coding concepts that will be used within the MMO game.</p>
+        {% for ks3_sessions in ks3_sheets %}
+            <table class="table--resources table--sessions">
+                {% for ks3_sheet in ks3_sessions %}
+                <tr>
+                    <td><p><strong>MMO</strong></p></td>
+                    <td><p>
+                        <a href="{% url 'materials_viewer' pdf_name=ks3_sheet.url_name %}">
+                            {{ ks3_sheet.title }} {% if ks3_sheet.video %}(includes video){% endif %}</a></p></td>
+                    <td><img title="PDF" alt="PDF icon" src='{% static "portal/img/icon_pdf.png" %}'></td>
+                    <td>
+                        {% if ks3_sheet.video %}
+                        <img title="Video" alt="Video icon" src='{% static "portal/img/icon_video.png" %}'>
```

Comment:
Can we make these images SVG's?

Reply:
yup

Reply:
we'll start with the ones specific to this PR - then I'll open an issue for the other images